cmd/podman/containers_prune.go

```diff
@@ -49,11 +49,11 @@ func pruneContainersCmd(c *cliconfig.PruneContainersValues) error {
 		reader := bufio.NewReader(os.Stdin)
 		fmt.Printf(`WARNING! This will remove all stopped containers.
 Are you sure you want to continue? [y/N] `)
-		ans, err := reader.ReadString('\n')
+		answer, err := reader.ReadString('\n')
```

Comment:
I really don't like this. Can we restrict to comments only? I don't want a spell-checker telling me what my variable names can and cannot be.

Reply:
Sure, and we can tell the codespeller to ignore certain names, in this case I actually liked the change, since I was not crazy about ans.